app/pods/publications/index/template.hbs

```diff
@@ -1,296 +1,293 @@
 {{!-- TODO auk-panel-layout gebruiken, component ? --}}
```

Comment:
Mag ik je in 1 moeite vragen om deze comment even te reviseren? Is dit nog relevant? Begrijp jij wat er bedoeld wordt? Doe hem anders gerust weg ...

Reply:
Volgens mij niet meer relevant (aangezien auk-panel niet gebruikt wordt/zal worden binnen deze pagina) dus comment verwijderd (+ nog even vlug onnodige spacer class van titel gehaald).